app/src/main/java/org/fossasia/openevent/app/common/data/repository/EventRepository.java

```diff
@@ -94,4 +99,19 @@ public EventRepository(IUtilModel utilModel, IDatabaseRepository databaseReposit
             .build();
     }
 
+    @NonNull
+    @Override
+    public Observable<Event> updateEvent(Event event) {
+        if (!utilModel.isConnected()) {
+            return Observable.error(new Throwable(Constants.NO_NETWORK));
+        }
+
+        event.setTickets(null);
```

Comment:
```Java
// Added to ignore serialization
    @JsonIgnore
    @ColumnIgnore
    @Relationship("tickets")
    public List<Ticket> tickets;

    @ColumnIgnore
    public static final String STATE_DRAFT = "draft";
    @ColumnIgnore
    public static final String STATE_PUBLISHED = "published";

    public Event() { }

    @JsonIgnore
    public List<Ticket> getTickets() {
        return tickets;
    }

    // Setter is annotated for enabling deserialization
    @JsonProperty
    public void setTickets(List<Ticket> tickets) {
        this.tickets = tickets;
    }
```

Also there should be proper response to user when the event update is failed or toggling. Not just setting the state back. Also, there is no indicator in Event list denoting the draft events